modules/docs/mdx/10.0-UPGRADE_GUIDE.mdx

```diff
@@ -123,6 +128,22 @@ these sizes should change along with it. Since `px` are a fixed unit and do not
 
 ## Component Updates
 
+### Buttons
+
+We re-factored how we styled Buttons to use our `createStyles` utility function. We don't anticipate
+this as a breaking change but, there may be slight changes to visual test.
+
+**PR:** [#2285](https://github.com/Workday/canvas-kit/pull/2285)
+
+#### Button Focus Ring Update
+
+We found that focus ring was not consistent across all Buttons. We have made some changes to create
+a consistent focus ring across all Buttons and variants. These will need to be taken note of due to
+some small changes visually with Buttons.
+
+- `PrimaryButton`: `inverse` variant now has the same focus ring as default variant and as
+  `SecondaryButton`.
+
```

Comment:
We should highlight the breaking change that `colors` no longer supports the `focusRing` option.

```jsx
import {focusRing} from '@workday/canvas-kit-react/common';

// before
<PrimaryButton
  colors={{
    // other colors
    focus: {
      // other colors
     focusRing: focusRing(/* options */)
    }
  }}
/>

// after
<PrimaryButton
  colors={{
    // other colors
    focus: {
      // other colors
    }
  }}
  css={{
    ':focus-visible': focusRing(/* options */
  }}
/>
```

Reply:
Done.